cmsranking/static/TimeView.js

```diff
@@ -92,7 +82,7 @@ var TimeView = new function () {
         }
 
         if (c == null) {
-            $("#TimeView_name").text();
+            $("#TimeView_name").text("");
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["$" is not defined.](https://www.codacy.com/app/s-maggiolo/cms/file/2192941803/issues/source?bid=3167167&fileBranchId=3167185#l85)